src/utils/i18n.js

```diff
@@ -0,0 +1,50 @@
+import i18next from "i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+export const I18N_ATTRIBUTE = "data-i18n";
+
+export const translateContent = (element) => {
+  const selectedDomTree = element || document;
+  const elements = selectedDomTree.querySelectorAll(`[${I18N_ATTRIBUTE}]`);
+  elements.forEach((el) => {
+    const key = el.dataset.i18n;
+    const t = i18next.t(key);
+    console.log(t, key);
+    // If translation found set element content as translation.
+    if (t !== key) el.innerHTML = t;
+    // Otherwise use english as fallback
+    else el.innerHTML = i18next.t(key, { lng: "en" });
+  });
+};
+
+// List of all languages we have translations for.
+const whitelist = ["en", "pt-BR"];
```

Comment:
Maybe move this to `constants.js` or move to the top of this file.  Just a suggestion.